### DISCUSSION TITLE
how to disable avx512f on ubuntu-latest?

### DISCUSSION THREAD

**Discussion by CodingSuen** [actions/virtual-environments#5734] [General questions] [answered]
My workflow is building the application on ubuntu-latest then ssh the tar to ecs server and deploy it, but the application did not work on ecs server. I find the reason the cpu of ecs server does not support avx512f instruction. Is it possible to disable avx512f on ubuntu-latest(github action) ?

**Comment by al-cheb** [comment, marked as answer]
It's not possible to disable avx512 CPU hardware instruction. You have to disable it in the codes.

**Reply by CodingSuen** [reply]
Thanks : )
